xds/internal/xdsclient/tests/ads_stream_ack_nack_test.go

```diff
@@ -337,9 +337,9 @@ func (s) TestADS_NACK_InvalidFirstResponse(t *testing.T) {
 //  1. A resource is requested and a good response is received. The test verifies
 //     that an ACK is sent for this resource.
 //  2. The previously requested resource is no longer requested. The test
-//     verifies that a request with no resource names is sent out.
+//     verifies that the connection to the management server is closed.
 //  3. The same resource is requested again. The test verifies that the request
-//     is sent with the previously ACKed version.
+//     is sent with an empty version string.
```

Comment:
it looks like you are verifying this by ensuring that the re-request is identical to first request. We should mention that here?